rta/recycle_bin_process.py

```diff
@@ -39,7 +37,7 @@ def main():
     common.log("Execute files from the Recycle Bin")
     target_dir = None
     for recycle_path in RECYCLE_PATHS:
-        if os.path.exists(recycle_path):
+        if Path(recycle_path).is_file() or Path(recycle_path).is_dir():
```

Comment:
Did you mean this?

```suggestion
        if Path(recycle_path).exists():
```

Reply:
Yes, will update to `.exists()`. It will work because `get_path` already is returning a string of a `PurePath` object, which uses a local directory if a full path is not passed. Since this RTA was written with this logic already in place it will not change the its functionality. However, if it was not a Path object, we may have to be careful with using `.exists()` as its functionality is not one to one with `os.path.exists`
```python
def get_path(*path: str) -> str:
    return str(Path(BASE_DIR).joinpath(*path))
```